### PR TITLE
fix a couple of typos in the metrics specification

### PR DESCRIPTION
**Pull request by Ladicek**
Signed-off-by: Ladislav Thon <lthon@redhat.com>

### CODE REVIEW

File: spec/src/main/asciidoc/metrics.asciidoc [hunk 1]
```diff
@@ -104,21 +104,19 @@ a| * `method` - the fully qualified method name
 
 === Metrics added for `@CircuitBreaker`
 
-ft.circuitbreaker.calls.total{method=<name>, circuitBreakerResult=[success|failure|circuitBreakerOpen]}
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/metrics.asciidoc:107]: Why did you delete this line?

Comment (Ladicek) [reply spec/src/main/asciidoc/metrics.asciidoc:107]: See how it looks in the rendered form. This line clearly isn't supposed to be there.

Comment (Azquelt) [reply spec/src/main/asciidoc/metrics.asciidoc:107]: Yeah, suspect I copied and pasted directly from the design in the issue as a placeholder before translating it to have the right formatting for the spec. In this case I forgot to take out the original placeholder.

**Comment by Ladicek** [comment]
Merging then, thanks.
